src/openrct2/rct2/T6Exporter.cpp

```diff
@@ -99,7 +99,10 @@ bool T6Exporter::SaveTrack(OpenRCT2::IStream* stream)
     {
         for (const auto& trackElement : _trackDesign->track_elements)
         {
-            tempStream.WriteValue<uint8_t>(trackElement.type);
+            int8_t trackType = trackElement.type;
+            if (trackElement.type == TrackElemType::RotationControlToggle)
```

Comment:
There is already a alias inside of TD6 format for vertical elements. We should be doing the conversion code for both all in one location.

Reply:
See line 214 of TrackDesign.cpp